app/agents/ApplicationAnalyzerAgent.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+import random
+import spade.behaviour
+
+from app.agents.RecruitmentManagerAgent import RecruitmentManagerAgent
+from app.dataaccess.model.JobOffer import ApplicationStatus
+from app.dataaccess.model.MessageType import MessageType
+from app.modules.JobOfferModule import JobOfferModule
+from app.utils.configuration import MASConfiguration
+
+from .base.BaseAgent import BaseAgent
+
+
+class ApplicationAnalyzerAgent(BaseAgent):
+    _instances: int = 0
+
+    def __init__(
+        self,
+    ):
+        ApplicationAnalyzerAgent._instances = ApplicationAnalyzerAgent._instances + 1
+        super().__init__(f"{ApplicationAnalyzerAgent._instances}")
```

Comment:
Czy `_instances` musi być zmienną statyczną? Może lepiej przekazywać takie "id" danej instancji `ApplicationAnalyzera` w konstruktorze (tj. w tej pętli w `main.py`, gdzie je startujesz można przekazać wartość iteratora)

Reply:
Zmieniłem to na swoim branchu całkowicie. Zamykam